app/frontend/i18n/i18n.ts

```diff
@@ -1491,6 +1491,7 @@ const options = {
             review_manager: "review manager",
             reviewer: "reviewer",
             super_admin: "super admin",
+            technical_support_local_jurisdiction: "Technical Support - LJ",
```

Comment:
you should probably just shorten this to technical_support for the sake of brevity and sanity

Reply:
This has been updated